web/ui/react-app/src/pages/starting/Starting.tsx

```diff
@@ -25,10 +25,10 @@ export const StartingContent: FC<StartingContentProps> = ({ status, isResponding
       <Logo height="400" />
       <div className="m-4">
         <h2>Starting up...</h2>
-        {status?.started && !status?.done ? (
+        {status?.started ? (
```

Comment:
I just changed this so the loading bar won't un render once the replaying is done, and instead will turn green.